### PR TITLE
fixed issue #18724 for 2.3-develop

### PR DESCRIPTION
**Pull request by Shubham0492**
### Description (*)

There was a issue in display tax percent if select any locale where comma(,) use instead of decimal (.) for an example Germany.
### Fixed Issues (if relevant)

1. #18724 : Display tax percent in current locale number format

### Manual testing scenarios (*)

1: login magento admin
2: Set in admin locale Germany or other European country where decimal character is comma
3: In Admin > Stores > Configuration > Sales > Tax > Shopping Cart Display Settings > Display Full Tax Summary = Yes and in the other section "Orders, Invoices, Credit Memos Display Settings"
4: Set a tax percent in admin with decimals like 50.1.
5: Add any product to cart and goto cart page.
6: Check order summary.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @Shubham-Webkul. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Tax/view/frontend/templates/order/tax.phtml [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 <td <?= /* @escapeNotVerified */ $block->getLabelProperties() ?>>
                     <?= $block->escapeHtml($title) ?>
                     <?php if (!is_null($percent)): ?>
-                        (<?= (float)$percent ?>%)
+                        (<?= /* @escapeNotVerified */ $_order->formatPrice((float)$percent) ?>%)
```
Comment (orlangur) [inline app/code/Magento/Tax/view/frontend/templates/order/tax.phtml]: Why not verified here?

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Tax/view/frontend/templates/order/tax.phtml (hunk 1 above)
Comment (Shubham0492) [reply app/code/Magento/Tax/view/frontend/templates/order/tax.phtml]: @orlangur  because it was reject in Travis CI process due to XSS vulnerability.

**Comment by Shubham0492** [comment]
@orlangur Please let me know if any changes required.

File: app/code/Magento/Tax/view/frontend/templates/order/tax.phtml (hunk 1 above)
Comment (orlangur) [reply app/code/Magento/Tax/view/frontend/templates/order/tax.phtml]: @Shubham-Webkul there should be another annotation, like `noEscape`, to indicate escaping is not needed and logic was verified.

Comment (Shubham0492) [reply app/code/Magento/Tax/view/frontend/templates/order/tax.phtml]: @orlangur I have updated in this commit https://github.com/magento/magento2/pull/19177/commits/6d60eb22e1afcd6ab4425cf0bcddde276ad0ac20

**Comment by Shubham0492** [comment]
@orlangur there is any changes required?

**Review by orlangur — APPROVED** [review]
@Shubham-Webkul thanks, looks great now!

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-3606](https://jira.corp.magento.com/browse/ENGCOM-3606) has been created to process this Pull Request

**Review by orlangur — DISMISSED** [review]
@Shubham-Webkul looks like `formatPrice` is not suitable here as it adds unnecessary dollar sign:

```
Problem: Percent value is shown with an unnecessary sign
Manual testing scenario:
1: Login to magento admin;
2: Set in admin locale Germany or other European country where decimal character is comma;
3: In Admin -> Stores -> Configuration -> Sales > Tax -> Shopping Cart Display Settings -> Display Full Tax Summary = Yes and in the other section "Orders, Invoices, Credit Memos Display Settings";
4: Set a tax percent in admin with decimals like 50.1;
5: Add any product to cart and go to “Proceed to Checkout”;
6: Fill all required fields and click Next;
7.Check order summary.
8.  In Tax tab percent value shown with an unnecessary sign ($).


Notice: The same problem occurs when:
 Place the order and see it in My Account

 See email sent about order 
```

Please fix implementation.

**Comment by sivaschenko** [comment]
Hi @Shubham-Webkul can you please update the branch and resolve conflicts so we can continue work on the pull request.

**Comment by orlangur** [comment]
Closing due to inactivity. Feel free to reach me out anytime later if you wish to continue work on this pull request and it will be reopened.
Thanks for collaboration!

**Pull request closed without merging**